packages/forms-web-app/src/config.js

```diff
@@ -65,5 +65,8 @@ module.exports = {
 	},
 	featureFlag: {
 		googleTagManager: process.env.FEATURE_FLAG_GOOGLE_TAG_MANAGER === 'true'
-	}
+	},
+	accessibilityLinkUrl:
+		`${process.env.HOST_URL}/accessibility-stamenent` ||
```

Comment:
typo

Reply:
Thank you Ed

Reply:
This can be `${server.host}/accessibility-statement`

Reply:
server.host points to localhost:3000 instead of localhost:9003 so had to work around that, and I don't think we can reference an object during initialization when using object literal syntax so that I can use server.host.
I need to have access to the dynamically changing base URL depending on the ENV, which is accomplished by the HOST_URL env variable.